lib/Cake/Test/Case/Model/Datasource/Database/MysqlTest.php

```diff
@@ -3368,6 +3376,26 @@ public function buildColumnUnsignedProvider() {
 					'default' => 1
 				),
 				'`testName` decimal UNSIGNED DEFAULT 1'
+			),
+			//set #9
```

Comment:
Instead of comments, you can use the array keys for documenting the data set.
The advantage is that it would show the name of the data set in case of an error in the error message.
It might even show it in the coverage, not sure though.